src/index.html

```diff
@@ -20,8 +20,504 @@
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main
+      class="container"
+      data-qa="card"
+    >
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__photo">
+          <img
+            class="card__image"
+            src="/src/images/imac.jpeg"
+            alt="imac"
+          />
+        </div>
+
+        <div class="card__title">
+          <p>APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+        </div>
+
+        <div class="card__subtitle">
+          <p>Product code: 195434</p>
+        </div>
+
+        <div class="card__feedback">
+          <div class="stars">
+            <div class="stars__star stars__stars--1"></div>
+            <div class="stars__star stars__stars--2"></div>
+            <div class="stars__star stars__stars--3"></div>
+            <div class="stars__star stars__stars--4"></div>
+            <div class="stars__star stars__stars--5"></div>
+          </div>
+
+          <div class="card__feedback-review">
+            <p>Reviews: 5</p>
+          </div>
+        </div>
+
+        <div class="card__price">
+          <div class="card__price card__price--title">
+            <p>Price:</p>
+          </div>
+
+          <div class="card__price card__price--amount">
+            <p>$2,199</p>
+          </div>
+        </div>
+
+        <div class="card__btn">
+          <a
+            data-qa="card-hover"
+            href="#"
+            class="card__button"
+          >
+            Buy
+          </a>
+        </div>
+      </div>
```

Comment:
An empty line here would be good also, for the same reason of my previous comment, these 2 cards are siblings